.github/workflows/test_markdown.yml

```diff
@@ -0,0 +1,33 @@
+# yamllint --format github .github/workflows/test_markdown.yml
+---
+name: test_markdown
+
+on:  # yamllint disable-line rule:truthy
```

Comment:
TIL why on was making truthy warnings.. here are some details @openzipkin/devops-tooling https://github.com/adrienverge/yamllint/issues/430

now, finally we have one workflow that has no lint output